2016/presentations/Rozklady_odwrotnie_potegowe/text.md

```diff
@@ -0,0 +1,34 @@
+## Przykłady
+Nasza ulubiona Wikipedia prezentuje wiele przykładów występowania rozkładów odwrotnie potęgowych z wielu dziedzin. Można zacząć od słowa, a dokładnie od częstotliwości jego występowania w długich tekstach. Jeśli policzymy, ile razy pojawiają się poszczególne słowa i wyznaczymy histogram to zobaczymy, że jego kształt będzie przypominał nasze rozkłady odwrotnie potęgowe. Przykładami z astronomii oraz geologii są ziarenka piasku oraz meteoryty. W świecie finansów jest to wielkość odszkodowań ubezpieczyciela za wypadki komunikacyjne, kto wie, może z użyciem tych rozkładów liczone są nasze składki OC i AC. Jako przykład bliższy tematyce Pycona możemy podać wielkość plików przesyłanych protokołem TCP. W tytułowym pytaniu wyliczone zostały przykłady z dziedziny geofizyki, która obfituje w rozkłady odwrotnie potęgowe. Tu warto zajrzeć do artykułu *Fitting and Goodness-of-Fit Test of Non-Truncated and Truncated Power-Law Distributions*[1] gdzie przedstawiono szersze problemy rozkładów oraz przykładowe dane. Cyklony tropikalne, a raczej ich energia, jak wszystkie wcześniej wymienione zjawiska charakteryzuje się tym, że duże zdarzają się bardzo rzadko. Myślę, że mieszkańcy Hawajów bardzo cieszą się z tego powodu. Pewnie to również wpływa na niewielką migrację ludzi z miejsc zagrożonych. Podobnie jest z lawinami błotnymi. Jednak najlepszym przykładem rozkładów odwrotnie potęgowych są trzęsienia ziemi. Przesuwające się płyty tektoniczne podczas zmian w naprężeniach dość często uwalniają  energię. Nie mniej jednak, są to zjawiska nieodczuwalne dla człowieka ze względu na ich wielkość. Jednak czasem głęboko pod ziemią występują zdarzenia o ogromnej energii powodujące fale sejsmiczne, które wywołują trzęsienia ziemi oraz inne katastrofy takie jak tsunami. Pewnie każdy słyszał, jak wielkie zniszczenia niosą ze sobą takie zjawiska, może dlatego też wielu naukowców próbuje zrozumieć ich działanie lub też wyliczyć prawdopodobieństwo ich wystąpienia. Rozkładem odwrotnie potęgowym możemy opisać moment sejsmiczny (energia) do częstości występowania, co wiąże się z bardzo dobrze znanym prawem Gutenberg-Richter, co widać na wzorze oraz wykresach. W 2002 roku udało się dość dobrze wyznaczyć wykładnik dla trzęsień ziemi, którego wartość wynosi blisko a=1.63. Co ciekawe dla tego zjawiska jest też drugi przykład rozkładu odwrotnie potęgowego, czyli czasy między trzęsieniami ziemi. Jest to związane z prawem Omori które mówi że trzęsienia ziemi potrafią wywoływać kolejne trzęsienia ziemi, tak zwane after shocki.
```

Comment:
"co wiąże się [...] co widać" -- to trzeba jakoś inaczej napisać, bo się powtarza. może "który wiąże się"?